docs/src/pages/demos/text-fields/ComposedTextField.js

```diff
@@ -20,6 +20,8 @@ const styles = theme => ({
 });
 
 class ComposedTextField extends React.Component {
+  labelRef = null;
```

Comment:
The bad flow times are back, adding code that does nothing just to make the type checker happy.

Reply:
Knowing something does nothing, got it.